### PR TITLE
terminololgy and phrasing in Introduction

### PR DESCRIPTION
**Pull request by henkbirkholz**
Rather extensive overhaul. Please check!

### CODE REVIEW

File: draft-ietf-lamps-csr-attestation.md [hunk 1]
```diff
@@ -84,45 +84,42 @@ for inclusion in PKCS#10 or CRMF certificate signing requests: an ASN.1 Attribut
...
+The term "attestation" is intentionally not defined in RFC 9334, but it is often taken to mean the activity of producing and appraising remote attestation Evidence.
+A Relying Party may consult that Evidence -- or an Attestation Result produced by a Verifier that has appraised the Evidence -- in making policy decisions about the trustworthiness of the
+target environment being assessed via appraisal of Evidence. {{architecture}} provides the basis to illustrate in this document how the various roles
 in the RATS architecture map to a certificate requester and a CA/RA.
 
 
 At the time of writing, several standard and several proprietary attestation technologies
 are in use.
-This specification thereby tries to be technology-agnostic with regards to the attestation technologies and instead focused on the transport of Evidence.
+This specification thereby is intended to as technology-agnostic as it is feasible with respect to implemented remote attestation technologies. This specification focuses on the transport of Evidence via CSR while making minimal assumptions about content or format of the transported Evidence.
```
Comment (henkbirkholz) [inline draft-ietf-lamps-csr-attestation.md:101]:
```suggestion
This specification thereby is intended to be as technology-agnostic as it is feasible with respect to implemented remote attestation technologies. This specification focuses on the transport of Evidence via CSR while making minimal assumptions about content or format of the transported Evidence.
```

File: draft-ietf-lamps-csr-attestation.md [hunk 2]
```diff
@@ -84,45 +84,40 @@ for inclusion in PKCS#10 or CRMF certificate signing requests: an ASN.1 Attribut
...
+This specification provides two newly defined attributes that allow for conveyance of remote attestation payload in Certificate Requests (CSR) in either PKCS#10 {{RFC2986}} or Certificate Request Message Format (CRMF) {{RFC4211}}.
 
 As outlined in the RATS Architecture {{RFC9334}}, an Attester (typically
-a device) produces a signed collection of evidence about its running environment.
-The term "attestation" is intentionally not defined in RFC 9334, but it is often taken to mean the overall process of producing and verifying evidence.
-A Relying Party may consult that evidence, or an attestation result produced by a verifier who has checked the evidence, in making policy decisions about the trustworthiness of the
-target environment being attested. {{architecture}} overviews how the various roles
+a device) produces a signed collection of Claims that constitutes Evidence about its running environment.
+The term "attestation" is intentionally not defined in RFC 9334, but it is often taken to mean the activity of producing and appraising remote attestation Evidence.
```
Comment (henkbirkholz) [inline draft-ietf-lamps-csr-attestation.md:93]: While RFC9334 does not define `attestation`, https://www.ietf.org/archive/id/draft-ietf-rats-tpm-based-network-device-attest-14.html#name-terminology-2 does. Should we take that into account?

File: draft-ietf-lamps-csr-attestation.md [hunk 3]
```diff
@@ -84,45 +84,40 @@ for inclusion in PKCS#10 or CRMF certificate signing requests: an ASN.1 Attribut
...
-may be necessary to validate evidence. The second Attribute carries a
-structure that may be used to carry evidence.
+may be required to validate signed Evidence. The second Attribute carries a
+structure that may be used to convey Evidence.
 
-A CSR may contain one or more evidence, for example evidence
-asserting the storage properties of the private key as well evidence
-asserting the firmware version and other general properties
-of the device, or evidence signed by certificate chains
-on different cryptographic algorithms.
```
Comment (henkbirkholz) [inline draft-ietf-lamps-csr-attestation.md:119]: As I assumed that `different cryptographic algorithms` does not really add anything meaningful here, I removed it. Did I unintentionally remove meaning here?

**Comment by hannestschofenig** [comment]
It is not a good idea to change the introduction when there is already another PR open that modifies the same text. This will result in lots of merge conflicts.

Additionally, there are open issues that need to be resolved and a re-write of the introduction is currently not an open issue. Note that the introduction has already been re-written several times.

**Comment by henkbirkholz** [comment]
The language used was deviating significantly from RFC 9334 terminology and RFC 4949 terminology. The conflict with #22 was only about capitalization, which is noted in this PR. Rebasing it is not a problem. I'll file an issue.

**Comment by hannestschofenig** [comment]
Where does the terminology conflict with  RFC 9334 and RFC 4949 terminology?

**Comment by henkbirkholz** [comment]
A few prominent examples:

* Examples of major conflicts
    * w.r.t. RFC 9334, `This evidence is to be verified by a relying party`: "verified" means nothing in the RATS scope and Evidence is never appraised by a Relying Party - only by Verifiers
    * w.r.t. RFC 4949, `certificate chains` is a term deprecated at least for 15 years and `certification path` should be used.
* Examples of minor conflicts
    * `evidence signed by certificate chains` This makes no sense. An Attester signes Evidence and it probably should sign with appropriate key material that is endorsed via certification paths
    * to stay in sync with RFC 9334 (colloquial) language, it is preferable to write that RATS conceptual messages are `conveyed` and not `carried`

**Comment by hannestschofenig** [comment]
I created two PRs to address the issues you have raised above instead of re-writing the text altogether. 

Three comments:

* Certificate chains is not a depreciated term. It depends what you want to say. 
* The terms "convey" and "carry" (or "transport") mean the same thing.
* The term "appraise" is something that aims to give the impression that there is something magic going on while in reality you are just comparing the provided data against some reference values. "Verification" is more natural.

**Comment by henkbirkholz** [comment]
These are only prominent examples and the text was littered with similar ones: I'd sincerely recommend to soldier through my proposal once instead of fixing each terminology nit one by one. I understand that reviewing takes time! But I think it is way more time consuming to comment on various related issues in several PRs. Like [here](https://github.com/lamps-wg/csr-attestation/pull/46#discussion_r1329808990) for example, where only some of the issues are addressed and a new issue is introduced.

File: draft-ietf-lamps-csr-attestation.md [hunk 4]
```diff
@@ -84,45 +84,40 @@ for inclusion in PKCS#10 or CRMF certificate signing requests: an ASN.1 Attribut
 
 # Introduction
 
-At the time that it is requesting a certificate from a Certification Authority (CA), a PKI end entity may wish to provide evidence of the security properties of the environment in which the private key is stored.
-This evidence is to be verified by a relying party such as the Registration Authority or the Certification Authority as part of validating an incoming certificate request against a given certificate policy.
-This specification provides a newly defined evidence attribute for carrying evidence in Certificate Requests (CSR) in either PKCS#10 {{RFC2986}} or Certificate Request Message Format (CRMF) {{RFC4211}}.
+When requesting a certificate from a Certification Authority (CA), a PKI end entity may wish to include Evidence of the security properties of its environments in which the private keys are stored in that request.
+This Evidence can be appraised by authoritative entities, such as a Registration Authority (RA) or a CA as part of validating an incoming certificate request against given certificate policies.
```
Comment (henkbirkholz) [inline draft-ietf-lamps-csr-attestation.md:88]:
```suggestion
This Evidence can be appraised by authoritative entities, such as a Registration Authority (RA) or a CA as part of validating an incoming certificate request against given certificate policy.
```

File: draft-ietf-lamps-csr-attestation.md [hunk 5]
```diff
@@ -84,45 +84,40 @@ for inclusion in PKCS#10 or CRMF certificate signing requests: an ASN.1 Attribut
 
 # Introduction
 
-At the time that it is requesting a certificate from a Certification Authority (CA), a PKI end entity may wish to provide evidence of the security properties of the environment in which the private key is stored.
-This evidence is to be verified by a relying party such as the Registration Authority or the Certification Authority as part of validating an incoming certificate request against a given certificate policy.
-This specification provides a newly defined evidence attribute for carrying evidence in Certificate Requests (CSR) in either PKCS#10 {{RFC2986}} or Certificate Request Message Format (CRMF) {{RFC4211}}.
+When requesting a certificate from a Certification Authority (CA), a PKI end entity may wish to include Evidence of the security properties of its environments in which the private keys are stored in that request.
```
Comment (henkbirkholz) [inline draft-ietf-lamps-csr-attestation.md:87]:
```suggestion
When requesting a certificate from a Certification Authority (CA), a PKI end entity may wish to include Evidence of the security properties of its environments in which private keys are stored.
```

File: draft-ietf-lamps-csr-attestation.md [hunk 6]
```diff
@@ -84,45 +84,40 @@ for inclusion in PKCS#10 or CRMF certificate signing requests: an ASN.1 Attribut
...
+This Evidence can be appraised by authoritative entities, such as a Registration Authority (RA) or a CA, or associated trusted Verifiers as part of validating an incoming certificate request against given certificate policies.
+This specification provides two newly defined attributes that allow for conveyance of remote attestation payload in Certificate Requests (CSR) in either PKCS#10 {{RFC2986}} or Certificate Request Message Format (CRMF) {{RFC4211}}.
```
Comment (hannestschofenig) [inline draft-ietf-lamps-csr-attestation.md:89]:
```suggestion
This specification defines an attribute and an extension that allow for conveyance of Evidence in Certificate Signing Requests (CSRs) in either PKCS#10 {{RFC2986}} or Certificate Request Message Format (CRMF) {{RFC4211}} payloads.
```

File: draft-ietf-lamps-csr-attestation.md [hunk 7]
```diff
@@ -84,45 +84,40 @@ for inclusion in PKCS#10 or CRMF certificate signing requests: an ASN.1 Attribut
...
+This specification defines an attribute and an extension that allow for conveyance of Evidence in Certificate Signing Requests (CSRs) in either PKCS#10 {{RFC2986}} or Certificate Request Message Format (CRMF) {{RFC4211}} payloads.
 
 As outlined in the RATS Architecture {{RFC9334}}, an Attester (typically
-a device) produces a signed collection of evidence about its running environment.
-The term "attestation" is intentionally not defined in RFC 9334, but it is often taken to mean the overall process of producing and verifying evidence.
-A Relying Party may consult that evidence, or an attestation result produced by a verifier who has checked the evidence, in making policy decisions about the trustworthiness of the
-target environment being attested. {{architecture}} overviews how the various roles
+a device) produces a signed collection of Claims that constitutes Evidence about its running environment.
+The term "attestation" is intentionally not defined in RFC 9334, but it is often taken to mean the activity of producing and appraising remote attestation Evidence.
```
Comment (hannestschofenig) [inline draft-ietf-lamps-csr-attestation.md:93]:
```suggestion
While the term "attestation" is not defined in RFC 9334, it was later defined in {{?I-D.[ietf-rats-tpm-based-network-device-attest}}, it refers to the activity of producing and appraising remote attestation Evidence.
```

File: draft-ietf-lamps-csr-attestation.md [hunk 8]
```diff
@@ -84,45 +84,40 @@ for inclusion in PKCS#10 or CRMF certificate signing requests: an ASN.1 Attribut
...
+A Relying Party may consult that Evidence -- or an Attestation Result produced by a Verifier that has appraised the Evidence -- in making policy decisions about the trustworthiness of the
```
Comment (hannestschofenig) [inline draft-ietf-lamps-csr-attestation.md:94]:
```suggestion
A Relying Party may consult an Attestation Result produced by a Verifier that has appraised the Evidence in making policy decisions about the trustworthiness of the
```

**Pull request merged**